### ISSUE TITLE
Compatibility Issues and Deprecation Warnings with Outdated @cypress/browserify-preprocessor

### ISSUE THREAD

**Issue by jasp402** [badeball/cypress-cucumber-preprocessor#1130]
### Current behavior

Issue: Compatibility Problems and Deprecation Warnings with `@cypress/browserify-preprocessor`
Hello,

I am encountering compatibility issues in my project due to the use of `@cypress/browserify-preprocessor`, which appears to be discontinued. During the installation of my dependencies, I am receiving deprecation warnings for certain Babel libraries, specifically:

```
deprecated @babel/plugin-proposal-class-properties@7.18.6
deprecated @babel/plugin-proposal-object-rest-spread@7.20.7
```

These warnings arise when installing the following dependencies:

```
"dependencies": {
  "@badeball/cypress-cucumber-preprocessor": "^18.0.6",
  "@cypress/browserify-preprocessor": "^3.0.2",
  "cypress": "^13.5.1",
  "multiple-cucumber-html-reporter": "^3.5.0"
},
"devDependencies": {
  "fs-extra": "^11.1.1",
  "standard-version": "^9.5.0"
}
```
I have noticed that the `@cypress/browserify-preprocessor` dependency is not only outdated (last version **3.0.2** published over **2 years ago**) but also its repository on GitHub has been archived, making the implementation of improvements impossible.

### Desired behavior

I would like to know if there is any recommendation on how to handle these deprecation warnings and if there is an alternative to `@cypress/browserify-preprocessor` that is more up-to-date and maintains the necessary functionality for my project.
I appreciate any advice or guidance you can provide to resolve this issue.

### Versions

* **Cypress version**: 13.6.0
* **Preprocessor version**: 19.1.1
* **Node version**: 20.9.0

### Checklist

- [x] I've read the [FAQ](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/faq.md).
- [x] I've read [instructions for logging issues](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/CONTRIBUTING.md#bug-reports).
- [x] I'm not using `cypress-cucumber-preprocessor@4.3.1` (package name has changed and it is no longer the most recent version, see [#689](https://github.com/badeball/cypress-cucumber-preprocessor/issues/689)).

**Comment by badeball** [comment]
https://github.com/badeball/cypress-cucumber-preprocessor/tree/master/examples

TLDR: just use esbuild.

**Comment by jasp402** [comment]
Now I have problems with trying to use **esbuild-cjs** the error appears 
```Error: ENOENT: no such file or directory, stat...```
 I have tried to perform the options that propose cleaning the cache etc. however it is not corrected. 😞

**Comment by badeball** [comment]
If you're trying to solicit help, then you need to be aware of the guidelines for [reporting issues](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/CONTRIBUTING.md#bug-reports).
